### PR TITLE
Added testing and infra for Velocity SNS tests

### PR DESCRIPTION
**Pull request by rethink-imcmahon**

### CODE REVIEW

**Comment by rethink-imcmahon** [comment]
> Does importing a single file which uses the BSD license make the whole package change licenses?
Not sure, I can change it back, but it likely doesn't matter.
> Having vals and vels so close in name makes this hard to read. Can you change vels to limits which would also make it clear what vels referred to?
Can you point out where you're referring to? Perhaps comment inline.
> Minor style: add spaces for readability.
> total_time += elapsed;
Yup. That's fine.
> The velocity tolerance should be a configuration parameter.
Agreed. This is a WIP, but there's no question that needs to be a param.

File: sns_ik_examples/src/ik_tests.cpp [hunk 1]
```diff
@@ -41,6 +43,19 @@ double fRand(double min, double max)
   return min + f * (max - min);
 }
 
+bool in_vel_bounds(KDL::JntArray vals, KDL::JntArray vels)
```
Comment (rethink-forrest) [inline sns_ik_examples/src/ik_tests.cpp]: Sorry, I thought the comments default to being inline. Here is where vals and vels were confusing.

Comment (rethink-forrest) [inline sns_ik_examples/src/ik_tests.cpp]: Sorry, I thought the comments defaulted to being inline. Here is where vals and vels were confusing.

**Comment by rethink-forrest** [comment]
I will merge this into master now since there will be a future commit to clean up the code.

**Pull request merged**